docs/Operators.md

```diff
@@ -4511,8 +4511,12 @@ opset_import {
 ### <a name="Split"></a><a name="split">**Split**</a>
 
   Split a tensor into a list of tensors, along the specified
-  'axis'. Lengths of the parts can be specified using argument 'split'.
-  Otherwise, the tensor is split to equal sized parts.
+  'axis'. Either 'split' or 'num_split' must be defined. If 'split'
```

Comment:
This is breaking change, you should bump the version number.

Reply:
Yes. Done.